com.unity.robotics.urdf-importer/Runtime/AssetHandlers/UrdfAssetPathHandler.cs

```diff
@@ -90,6 +90,7 @@ public static string GetFullAssetPath(string relativePath)
         public static string GetRelativeAssetPathFromUrdfPath(string urdfPath, bool convertToPrefab=true)
         {
             string path;
+            bool use_file_uri = false;
```

Comment:
style for this file is camelCase for local variables: `useFileUri`

Reply:
I fixed in https://github.com/Unity-Technologies/URDF-Importer/pull/147/commits/399cec04a3658d587c441b25db33a8df93d669fc

Please check!